packages/vscode-extension/src/controls/SampleGallery.tsx

```diff
@@ -3,6 +3,7 @@ import { Icon, Stack, Image, PrimaryButton } from '@fluentui/react'
 import './SampleGallery.scss'
 import { Commands } from './Commands'
 import HelloWorld from '../../media/helloworld.gif'
```

Comment:
should we need to remove "HelloWorld"

Reply:
Yes, after all resources done. But currently only in meeting app is ready. Other apps still need a placeholder